pkgs/servers/sql/patroni/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, pythonPackages, fetchurl }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "patroni-${version}";
+  version = "1.6.4";
+
+  src = fetchurl {
+    url = "https://github.com/zalando/patroni/archive/v${version}.tar.gz";
+    sha256 = "0a8pzy3sf164byz4wkjcgm6yi45y2rkaf8jvzlhnbq3rdq68pb0x";
+  };
+
+  postPatch = ''
+    # cdiff renamed to ydiff; remove when patroni source reflects this.
+    for i in requirements.txt patroni/ctl.py tests/test_ctl.py; do
+      substituteInPlace $i --replace cdiff ydiff
+    done
+  '';
+
+  nativeBuildInputs = with pythonPackages; [
```

Not a big deal, but I'd use `checkInputs` instead:

```suggestion
  checkInputs = with pythonPackages; [
```